client/pages/sections/protocols/section.js

```diff
@@ -27,15 +29,15 @@ class Section extends PureComponent {
           editable && !values.deleted
             ? (
               <Fieldset
-                fields={fields}
+                fields={fields.filter(f => f.show === undefined || f.show(project))}
                 values={values}
                 prefix={prefix}
                 onFieldChange={onFieldChange}
               />
             )
             : (
               <ReviewFields
-                fields={fields}
+                fields={fields.filter(f => f.show === undefined || f.show(project))}
```

Comment:
This should probably move inside the component and be handled there, but that's probably a thing for another PR.

Reply:
In fact, it looks like it already _is_ handled in `ReviewFields`

https://github.com/UKHomeOffice/asl-projects/blob/master/client/components/review-fields.js#L10-L18